Bugzilla/Config/Auth.pm

```diff
@@ -17,6 +17,14 @@ use Types::Common::Numeric qw(PositiveInt);
 
 our $sortkey = 300;
 
+use constant PASSWDQC_DESC => <<'DESC';
+Password containing only one class of characters is not allowed.<br>
+Password containing two classes of characters must be 24 characters or longer<br>
+Password that is a passphrase must be 11 characters or longer, and at least 3 words.<br>
+Password containing three classes of characters must be 8 characters or longer.<br>
+Password containing four classes of characters must be 7 characters or longer.
```

Comment:
this doesn't describe what a "class of character" is.

i'm concerned there's a lot of dense information here; although a lot of it should be not required with `undef,undef,12,12,12`.

perhaps it would be better to have a simpler "dumbed down" explanation here, which provides enough of a hint for most users, with a link to a wiki page that contains the full technical breakdown for each character class.
eg.

> Your password must be at least <b>12 characters long</b>, and contain:
> * a mixture of uppercase, lowercase, and numbers
> OR
> * a mixture or symbols, numbers, and letters
> OR
> * a phrase with at least four words
> 
> <a href="#">View our full password complexity rules.</a>

Reply:
I changed the default message to something terse. The revised message will be part of configuration.